### PR TITLE
PD-213 Update 22.12.2 Release Notes

### PR DESCRIPTION
**Pull request by micjohnson777**
This PR updates the 22.12.2 release notes merged into the 22.2.2_docs branch last Friday.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALE22.12.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ It also implements fixes to pool status reporting, application options, reportin
 ### Component Versions
 
 TrueNAS SCALE is built from many different software components.
-This list has up to date information on which versions of Linux, ZFS, and NVIDIA drivers are included with this TrueNAS SCALE release.
+This list has up to date information on which versions of Linux, ZFS, and NVIDIA drivers included with this TrueNAS SCALE release.
```
Comment (tonyriv3) [inline content/SCALE/SCALE22.12.md:82]:
```suggestion
This list has up-to-date information on which versions of Linux, ZFS, and NVIDIA drivers are included with this TrueNAS SCALE release.
```

**Review by tonyriv3 — DISMISSED** [review]

File: content/SCALE/SCALE22.12.md (hunk 1 above)
Comment (micjohnson777) [reply content/SCALE/SCALE22.12.md:82]: fixed in my local copy to push in next commit.

File: content/SCALE/SCALE22.12.md [hunk 2]
```diff
@@ -1508,38 +1508,42 @@ Known issues are those found during internal testing or reported by the communit
...
 | 22.12.2 | <a href="https://ixsystems.atlassian.net/browse/NAS-121014" target="_blank">NAS-121014</a> | System configured on LA time but Network dashboard card is in EST | System time configured PST while the main Dashboard Network card displays EST. | 22.12.3<br> 23.10-ALPHA.1 Cobia |
-| 22.12.2 | <a href="https://ixsystems.atlassian.net/browse/NAS-120961" target="_blank">NAS-120961</a> | HA: Connected TrueCommand Cloud status is Disabled after HA Failover | Add an HA system to TrueCommand, failover of the HA system results in TrueCommand cloud status change to Disabled after the failover. TrueCommand should remain enabled and connected. | 22.12.3 |
+| 22.12.2 | <a href="https://ixsystems.atlassian.net/browse/NAS-120961" target="_blank">NAS-120961</a> | HA: Connected TrueCommand Cloud status is disabled after HA failover | Add an HA system to TrueCommand, failover of the HA system results in TrueCommand cloud status change to Disabled after the failover. TrueCommand should remain enabled and connected. | 22.12.3 |
```
Comment (tonyriv3) [inline content/SCALE/SCALE22.12.md:1535]:
```suggestion
| 22.12.2 | <a href="https://ixsystems.atlassian.net/browse/NAS-120961" target="_blank">NAS-120961</a> | HA: Connected TrueCommand Cloud status is disabled after HA failover | If you add an HA system to TrueCommand and perform a failover, TrueCommand cloud disables. TrueCommand should remain enabled and connected. | 22.12.3 |
```

**Review by tonyriv3 — CHANGES_REQUESTED** [review]

**Review by ericbsd — APPROVED** [review]
LGTM

**Pull request merged**